src/Codeception/Lib/Connector/Yii2/ConnectionWatcher.php

```diff
@@ -55,10 +56,10 @@ public function closeAll(): void
         }
     }
 
-    protected function debug($message): void
+    protected function debug(mixed $message): void
```

Comment:
Should we change this to `array|JsonSerializable`? Since we are calling `json_encode` on it anyway?